src/studySetup.js

```diff
@@ -34,19 +34,8 @@ const baseStudySetup = {
     removeTestingFlag: false,
   },
 
-  // endings with urls
-  endings: {
-    /** standard endings */
-    "user-disable": {
-      baseUrls: [],
-    },
-    ineligible: {
-      baseUrls: [],
-    },
-    expired: {
-      baseUrls: [],
-    },
-  },
+  // We don't have a survey at the ending
+  endings: { },
```

Comment:
This seems to effectively kill the "Disable Privacy Study" checkbox feature? On this branch, if I check the "Disable Privacy Study" checkbox when I submit a page broken ping, the add-on remains?

Reply:
yeah, I noticed that too, but I figured it's okay since I'm removing that in the next PR anyway.